snap/snapcraft.yaml

```diff
@@ -220,73 +192,11 @@ parts:
       'fastdds_no_shared_memory.xml': usr/share/
 
   cleanup:
-    after: [gazebo, ros2-foxy-extension]
+    after: [gazebo]
```

Comment:
gazebo doesn't have an `after: ros2-humble-extension` so here the priming of `ros2-humble-extension` could happen after the cleanup (which is not what we want)